wiki/Tournaments/MCNC/4K2022/en.md

```diff
@@ -0,0 +1,623 @@
+| **Quotient GD** | **0** | -1 | Yayoi Takatsuki | *Win by Default* |
+| **Kuiiiiteeee** | **4** | 0 | Hunxin | [#1](https://osu.ppy.sh/community/matches/102099391) |
+| **Ricizus** | **4** | 0 | zerda | [#1](https://osu.ppy.sh/community/matches/102102890) |
+| **H dame** | **0** | -1 | U1d | *Win by Default* |
+| **\[GB\]Mafufu** | **4** | 0 | luluxia | [#1](https://osu.ppy.sh/community/matches/102119450) |
+| **\[GB\]AelSan** | **4** | 0 | dreamed123 | [#1](https://osu.ppy.sh/community/matches/102121614) |
+| **\[Crz\]Riko** | **4** | 0 | ICspoon | [#1](https://osu.ppy.sh/community/matches/102122453) |
+| **\[Crz\]rebellion** | **4** | 0 | \[GB\]dotdot | [#1](https://osu.ppy.sh/community/matches/102123447) |
+| **\[GB\]gluefly** | **4** | 1 | MCPXiaoBai | [#1](https://osu.ppy.sh/community/matches/102124142) |
+| **Yana Feiya** | **4** | 0 | 2580697339 | [#1](https://osu.ppy.sh/community/matches/102125031) |
+| **alivn** | **4** | 0 | \[GB\]Cz | [#1](https://osu.ppy.sh/community/matches/102125996) |
+| **murorachi** | **4** | 0 | \[JbL\]Hx_032 | [#1](https://osu.ppy.sh/community/matches/102126003) |
+| **\[GB\]Luoxuan0327** | **4** | 0 | Sakurada Shiro | [#1](https://osu.ppy.sh/community/matches/102126977) |
+| Krn_ | -1 | **0** | **shizehao** | *Win by Default* |
+| **\[Crz\]IsxI** | **4** | 3 | CLLbin | [#1](https://osu.ppy.sh/community/matches/102127958) |
+| **\[Crz\]lf1f1f** | **4** | 0 | Etanity Larva | [#1](https://osu.ppy.sh/community/matches/102129139) |
+| \[GB\]yobrevelc | 2 | **4** | **\[GB\]sherweifa** | [#1](https://osu.ppy.sh/community/matches/102129090) |
+| **\[Crz\]sunnyxxy** | **4** | 0 | MIKUlove_ | [#1](https://osu.ppy.sh/community/matches/102130485) |
+| **Wings_AnChen** | **4** | 2 | gzdongsheng | [#1](https://osu.ppy.sh/community/matches/102130489) |
+| **rurudo1o1o** | **4** | 3 | Rudown | [#1](https://osu.ppy.sh/community/matches/102132006) |
+| **\[Crz\]Reimu** | **4** | 0 | iRedi | [#1](https://osu.ppy.sh/community/matches/102149048) |
+| **Akermo** | **0** | -1 | YuanLiang | *Win by Default* |
+| **tyrcs** | **4** | 0 | Gregcraft | [#1](https://osu.ppy.sh/community/matches/102150938) |
+| **Miyes** | **4** | 1 | N3k0ha_5h12uku | [#1](https://osu.ppy.sh/community/matches/102151683) |
+| **\[GB\]Tyris** | **4** | 1 | Molli | [#1](https://osu.ppy.sh/community/matches/102152440) |
+| **Zyuuu** | **4** | 0 | YUCInoback | [#1](https://osu.ppy.sh/community/matches/102154297) |
+| **- Minato  Aqua-** | **4** | 0 | shade0214 | [#1](https://osu.ppy.sh/community/matches/102155252) |
+| **chana** | **4** | 2 | MkoTenshi | [#1](https://osu.ppy.sh/community/matches/102155260) |
+| **YuLiangSSS** | **4** | 0 | katiexie | [#1](https://osu.ppy.sh/community/matches/102156436) |
+| **- Xiaoluoli -** | **4** | 1 | DJ Sharpnel | [#1](https://osu.ppy.sh/community/matches/102156358) |
+| **-Chen-** | **4** | 3 | Alptraum | [#1](https://osu.ppy.sh/community/matches/102157598) |
+
+### Qualifiers
```

Comment:
```suggestion
### Qualifiers
```
Inconsistent naming, if we're sticking with Line 20, then change it to Qualifier stage,
If we're sticking to Qualifiers, change the Line 20 one.